addons/sale_stock/models/stock.py

```diff
@@ -28,7 +28,7 @@ def _prepare_merge_move_sort_method(self, move):
 
     def _action_done(self):
         result = super(StockMove, self)._action_done()
-        for line in self.mapped('sale_line_id'):
+        for line in result.mapped('sale_line_id'):
```

Comment:
why? what's the usecase where before and after _action_done you would have different move?